data/profiles/default.yaml

```diff
@@ -10,458 +10,219 @@ spec:
-            - port: 15031
-              targetPort: 15031
-              name: grafana
-            - port: 15032
-              targetPort: 15032
-              name: tracing
-              # This is the port where sni routing happens
-            - port: 15443
-              targetPort: 15443
-              name: tls
-            #### MESH EXPANSION PORTS  ########
-            # Pilot and Citadel MTLS ports are enabled in gateway - but will only redirect
-            # to pilot/citadel if global.meshExpansion settings are enabled.
-            # Delete these ports if mesh expansion is not enabled, to avoid
-            # exposing unnecessary ports on the web.
-            # You can remove these ports if you are not using mesh expansion
-            meshExpansionPorts:
-            - port: 15011
-              targetPort: 15011
-              name: tcp-pilot-grpc-tls
-            - port: 8060
-              targetPort: 8060
-              name: tcp-citadel-grpc-tls
-            - port: 853
-              targetPort: 853
-              name: tcp-dns-tls
-            ####### end MESH EXPANSION PORTS ######
-            ##############
-            secretVolumes:
-            - name: ingressgateway-certs
-              secretName: istio-ingressgateway-certs
-              mountPath: /etc/istio/ingressgateway-certs
-            - name: ingressgateway-ca-certs
-              secretName: istio-ingressgateway-ca-certs
-              mountPath: /etc/istio/ingressgateway-ca-certs
-            # Telemetry addon gateways example config
-            telemetry_addon_gateways:
-              tracing_gateway:
-                name: tracing
-                port: 15032
-                desPort: 80
-                enabled: false
-                tls: false
-              kiali_gateway:
-                name: kiali
-                port: 15029
-                desPort: 20001
-                enabled: false
-                tls: false
-              grafana_gateway:
-                name: grafana
-                port: 15031
-                desPort: 3000
-                enabled: false
-                tls: false
-              prometheus_gateway:
-                name: prometheus
-                port: 15030
-                desPort: 9090
-                enabled: false
-                tls: false
+        enabled: true
+        k8s:
+          hpaSpec:
+            maxReplicas: 5
+            minReplicas: 1
+            scaleTargetRef:
+              apiVersion: apps/v1
+              kind: Deployment
+              name: istio-ingressgateway
+            metrics:
+            - type: Resource
+              resource:
+                name: cpu
+                targetAverageUtilization: 80
+          resources:
+            requests:
+              cpu: 100m
+              memory: 128Mi
+            limits:
+              cpu: 2000m
+              memory: 1024Mi
+
       egressGateway:
-        common:
-          enabled: false
-          k8s:
```

Comment:
Do we want to remove the k8s overlay part for `egressGateway`? 
Although it is disabled in `default` profile, but other profiles(`demo` and `demo-auth`) and customized API will need this.

Reply:
Good point, done.